test/Spec.hs

```diff
@@ -494,11 +498,11 @@ createResponseTicketSpec =
         it "should preserve tags from ticketinfo and zendeskresponse" $
             property $ \agentId ticketInfo zendeskResponse ->
                 let responseTicket      = createResponseTicket agentId ticketInfo zendeskResponse
-                    ticketInfoTags      = getTicketTags (tiTags ticketInfo)
-                    zendeskResponseTags = getTicketTags (zrTags zendeskResponse)
-                    mergedTags          = ticketInfoTags <> zendeskResponseTags
+                    mergedTags          = getTicketTags $ tiTags ticketInfo <> zrTags zendeskResponse
                     responseTags        = getTicketTags $ tTag responseTicket
-                in all (`elem` responseTags) mergedTags
+                -- in summary, the response tags have the debuggers `analyzed-by-script-version` tag AND
+                -- they remove the `to_be_analysed` tag AND they are unique.
+                in (filter (/= renderTicketStatus ToBeAnalyzed) . nub $ renderTicketStatus AnalyzedByScriptV1_3 : mergedTags) === responseTags
```

Comment:
The good thing about this is this will fail once we increase the tag version.

Reply:
Ah, thank you for letting me know! I'm be careful about it